secrets/secrets_test.go

```diff
@@ -2,11 +2,86 @@ package secrets
 
 import (
 	"fmt"
+	"sync"
 	"testing"
 
+	"github.com/checkmarx/2ms/plugins"
+	"github.com/checkmarx/2ms/reporting"
 	"github.com/zricethezav/gitleaks/v8/config"
 )
 
+func TestSecrets(t *testing.T) {
+	secrets := []struct {
+		Content    string
+		Name       string
+		ShouldFind bool
+	}{
```

Comment:
When you will update your branch with the `master` you will see that this test already exists at the end of the file, and you will need to merge the existing test with yours.